current/advanced/block-import.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: block-import
+lang: en
+title: The Block Import Pipeline
+---
+
+## The Import Queue
+
+The import queue is an abstract worker queue present in every Substrate node. It is not part of the
+runtime. The Import Queue is responsible for processing pieces of incoming information, verifying
+them, and if they are valid, importing them into the node's state. The most fundamental piece of
+information that the import queue processes is blocks themselves, but it is also responsible for
+importing consensus-related messages such as Justifications and, in light clients, finality proofs.
```

Comment:
Why is Justifications capitalized? Can you link to more information about Justifications and finality proofs?

Reply:
Removed the caps on justification.

It _was_ capitalized because it is this struct. https://crates.parity.io/sp_runtime/type.Justification.html I can link that if you think it's helpful to do so. There is no corresponding thing to link for finality proof.